client/src/pages/AdminLogin/index.jsx

```diff
@@ -127,7 +144,7 @@ const AdminLogin = () => {
           {passwordInputField}
           {submitLoginButton}
         </form>
-      </div>
+      </Paper>
```

Comment:
`AdminLogin` is >130 lines long; is there anything you can bring out of the function definition?

Reply:
Yes, moved all eventHandlers to file `eventHandlers.js

Reply:
Currently, everything must be inside the function as most of the handlers require dispatch. I attempted restructuring into a separate file for event handlers but it requires too much code overhead to not get working.